src/ui/features/referral-program/shared/ReferrerLink/ReferrerLink.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { getWalletDisplayName } from 'src/ui/shared/getWalletDisplayName';
+import { TextAnchor } from 'src/ui/ui-kit/TextAnchor';
+
+export function ReferrerLink({
+  handle,
+  address,
+  ...props
+}: React.AnchorHTMLAttributes<HTMLAnchorElement> & {
+  handle: string | null;
+  address: string;
+}) {
+  const displayName = getWalletDisplayName({
+    name: handle,
+    address,
+  });
```

Comment:
Why not use `<WalletDisplayName />` component?

Reply:
I don't already remember the reason I used `getWalletDisplayName` here instead of `<WalletDisplayName />`, lemme try to use the component

Reply:
Ah, that is because `<WalletDisplayName />` needs `wallet: ExternallyOwnedAccount` and we only have `name` (`handle`) and `address` here

Reply:
`ExternallyOwnedAccount` _is_ just `{ name, address }`